x-pack/plugins/canvas/canvas_plugin_src/functions/common/plot/get_font_spec.ts

```diff
@@ -26,8 +28,8 @@ export const getFontSpec = argFont => {
   const lHeight = typeof lineHeight === 'string' && Number(lineHeight.replace('px', ''));
 
   return {
-    size: !isNaN(size) ? size : defaultSpec.size,
-    lHeight: !isNaN(size) ? lHeight : defaultSpec.lHeight,
+    size: size && !isNaN(size) ? size : defaultSpec.size,
```

Comment:
Is it intended that a zero `size` (falsey) causes a fallback to `defaultSpec.size`?

Reply:
It's to handle a #wtfjs case, and not affect existing functionality:

![Screen Shot 2019-04-24 at 5 16 07 PM](https://user-images.githubusercontent.com/297604/57087822-e99db380-6cce-11e9-9f85-73d8dd971623.png)

Reply:
I suppose `Number.isNaN()` won't help either as it'll yield `false` on `undefined`... fun with JS types indeed!